wpilibj/src/athena/cpp/lib/HALUtil.cpp

```diff
@@ -52,7 +56,7 @@ static jclass pwmConfigDataResultCls = nullptr;
 
 namespace frc {
 
-static void GetStackTrace(JNIEnv *env, std::string &res, std::string &func) {
+static void GetStackTrace(JNIEnv *env, llvm::SmallString<128> &res, llvm::SmallString<128> &func) {
```

Comment:
The correct way to do this is to use llvm::SmallVectorImpl<char>, which doesn't encode the length in the parameter.